package.json

```diff
@@ -164,6 +164,7 @@
     "intersection-observer": "^0.7.0",
     "jsonlint-mod": "^1.7.5",
     "lodash": "^4.3.0",
+    "marked": "^1.2.2",
```

Comment:
i think we also want to run `yarn remove react-markdown` to get rid of the older library, right?

Reply:
Oh yes, forgot that i hadn't removed it. Good catch!